### ISSUE TITLE
Collpase branches of inheritance & call graphs

### ISSUE THREAD

**Issue by egraether** [CoatiSoftware/SourcetrailBugTracker#456]

**Comment by egraether** [comment]
Workaround since 2018.1: Hide nodes via context menu to remove branches

**Comment by ericlemasson** [comment]
Yes, feature like collapse / uncollapse per context menu per node would be nice. Hide option is just ok, but painful when want to remove all the node below a given node. And how to show again what's been hidden? I could not find any option "show hidden node", I did had to re-draw the graph.
